package.json

```diff
@@ -63,5 +63,8 @@
         "jest": "^24.5.0",
         "node-forge": "^0.7.6",
         "pdfkit": "^0.10.0"
+    },
+    "dependencies": {
+        "pdf-lib": "^1.2.1"
```

Comment:
The library does not have explicit dependencies. We use peer dependencieso that choosing your libs is up to you. We only bundle our code and rely on the user to provide the needed deps.